### PR TITLE
(Fix) Don't show addresses if tx from metamask was failed

### PR DESCRIPTION
**Pull request by rstormsf**
1. Dynamically load addresses based on chainId from metamask
2. Don't show mining,payout,voting keys if tx was failed

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -77,10 +78,11 @@ class App extends Component {
         swal("Congratulations!", "Your keys are generated!", "success");
       }).catch((error) => {
         console.error(error.message);
-        this.setState({loading: false})
+        this.setState({loading: false, keysGenerated: false})
         var content = document.createElement("div");
         content.innerHTML = `<div>
-          Something went wrong!<br/><br/> 
+          Something went wrong!<br/><br/>
+          Please contract Master Of Ceremony<br/><br/>
```
Comment (vbaranov) [inline src/App.js]: Typo: `contract` -> `contact`

File: src/addresses.js [hunk 2]
```diff
@@ -1,3 +1,21 @@
-module.exports = {
-  KEYS_MANAGER_ADDRESS: '0xfc90125492e58dbfe80c0bfb6a2a759c4f703ca8'
+const local = {
```
Comment (vbaranov) [inline src/addresses.js]: const `local` is assigned a value but never used

File: src/App.js (hunk 1 above)
Comment (rstormsf) [reply src/App.js]: 👍

File: src/addresses.js (hunk 2 above)
Comment (rstormsf) [reply src/addresses.js]: good catch!

**Review by vbaranov — APPROVED** [review]
Successfully tested in local environment

**Pull request merged**